### PR TITLE
Set the deault list style using config.

### PR DESCRIPTION
**Pull request by robonetphy**
includes #31 
includes #41

### CODE REVIEW

**Review by ilyamore88 — APPROVED** [review]
👍

File: src/index.js [hunk 1]
```diff
@@ -78,15 +78,16 @@ export default class NestedList {
...
-    this.defaultListStyle = 'ordered';
+    this.defaultListStyle = this.config.defaultStyle === 'unordered'  ? 'unordered' : 'ordered';
+
 
     const initialData = {
       style: this.defaultListStyle,
       items: [],
     };
-
+
```
Comment (talyguryn) [inline src/index.js:90]:
```suggestion

```

File: src/index.js [hunk 2]
```diff
@@ -78,15 +78,16 @@ export default class NestedList {
     this.config = config;
 
     /**
-     * This list-style will be used by default
+     * Set the default list style from the config.
      */
-    this.defaultListStyle = 'ordered';
+    this.defaultListStyle = this.config.defaultStyle === 'unordered'  ? 'unordered' : 'ordered';
```
Comment (talyguryn) [inline src/index.js:83]: please make `unordered` by default

**Review by talyguryn — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (robonetphy) [reply src/index.js:83]: Orderd is default in list tool too. That why i have put the same here

Comment (talyguryn) [reply src/index.js:83]: so let's update that tool too

Comment (robonetphy) [reply src/index.js]: okay

**Review by neSpecc — APPROVED** [review]

**Review by talyguryn — APPROVED** [review]

**Pull request merged**